jni/audioengine.cpp

```diff
@@ -328,23 +330,25 @@ namespace AudioEngine
                 // update the buffer pointers and sequencer position
                 if ( playing )
                 {
-                    if ( ++bufferPosition % bytes_per_tick == 0 )
-                       handleSequencerPositionUpdate( android_GetTimestamp( p ));
+                    if ( bufferPosition % samples_per_step == 0 )
+                       handleSequencerPositionUpdate( bufferPosition );
```

Comment:
now broadcasting bufferPosition via JNI, see MWEngineActivity to see how it is processed in Java.